src/clr-core/styles/tokens/tokens.yml

```diff
@@ -179,6 +179,66 @@ props:
     type: 'number'
     category: 'typography'
 
+  # global font sizes
+  - name: font-size-0
+    value: '10px'
+    type: 'px'
+    category: 'typography'
+  - name: font-size-1
+    value: '12px'
+    type: 'px'
+    category: 'typography'
+  - name: font-size-2
+    value: '14px'
+    type: 'px'
+    category: 'typography'
+  - name: font-size-3
+    value: '16px'
+    type: 'px'
+    category: 'typography'
+  - name: font-size-4
+    value: '20px'
+    type: 'px'
+    category: 'typography'
+  - name: font-size-5
+    value: '24px'
+    type: 'px'
+    category: 'typography'
+  - name: font-size-6
+    value: '32px'
+    type: 'px'
+    category: 'typography'
+  - name: font-size-7
+    value: '40px'
+    type: 'px'
+    category: 'typography'
+
+  # global line-heights
+  - name: line-height-0
+    value: 1
+    type: 'number'
+    category: 'typography'
+  - name: line-height-1
+    value: 1.1
+    type: 'number'
+    category: 'typography'
+  - name: line-height-2
```

Comment:
line-height should be in ems. I've run into problems using just the plain number (even though it's supposed to work as expected).

i also don't understand why these are necessary.

Reply:
I dont think they are necessary thinking about it. There are cases where we need sizes internally but line heights look like they will be more specific to the use case so I'll pull them out.